helm/prometheus-rules/templates/recording-rules/grafana-cloud.rules.yml

```diff
@@ -186,6 +186,10 @@ spec:
       record: aggregation:ingress:management_cluster_replicas
     - expr: sum(rate(nginx_ingress_controller_nginx_process_requests_total[5m])) by (cluster_id, cluster_type, customer, installation, pipeline, provider, region)
       record: aggregation:ingress:requests_total
+    {{- with .Values.Installation.V1.Guest.Kubernetes.IngressController.BaseDomain }}
+    - expr: count(count(kube_ingress_path{host=~".*{{ . }}"}) by (ingress,cluster_id)) by (cluster_id)
```

Comment:
Why is this double `count` necessary? `count(kube_ingress_path{host=~".*{{ . }}"}) by (cluster_id)` wont' do the same?

Reply:
No, note the metric is per path; so ingresses with more than 1 path will have many entries. 

- First count merges the paths for the same ingress in one label.
- Second count actually counts how many ingresses are in the cluster that at least have 1 path with the base domain.

Maybe there's another way to do this, but the query with 1 count will count paths per cluster_id, not ingresses.